front/src/modules/ui/table/components/EntityTable.tsx

```diff
@@ -88,6 +90,7 @@ type OwnProps = {
 
 export const EntityTable = ({ updateEntityMutation }: OwnProps) => {
   const tableBodyRef = useRef<HTMLDivElement>(null);
+  const [dragSelect, setDragSelect] = useRecoilState(dragSelectState);
```

Comment:
Could you find a more self-explanatory naming, it should tell us what's happening. 

Maybe `isDragSelecting` or `isDraggingAndSelecting` ?

Reply:
Thanks @lucasbordeau! I always struggle with naming variables :')